client/state/data-layer/wpcom-http/actions.js

```diff
@@ -1,12 +1,30 @@
 /**
  * Internal dependencies
  */
-import { WPCOM_HTTP_REQUEST } from 'state/action-types';
+import {
+	WPCOM_HTTP_REQUEST,
+	WPCOM_HTTP_BAD_REQUEST,
+} from 'state/action-types';
+import warn from 'lib/warn';
+
+export const apiVersionRegex = new RegExp( /^v(?:(\d+)\.)?(\d+)$/ );
+export const apiNamespaceRegex = new RegExp( /^(?:([a-z]+)\/)?v(\d+)$/ );
```

Comment:
there's no need to write `new Regexp` here if you don't want to. It doesn't offer any advantage over creating a RegExp literal `/something/`